### PR TITLE
Improvements on sample clients

### PR DESCRIPTION
**Pull request by RAVEENSR**
## Purpose
Improvements on sample clients

## Goals
Set error free sample clients

## Approach
N/A

## User stories
N/A

## Release note
N/A

## Documentation
N/A

## Training
N/A

## Certification
N/A

## Marketing
N/A

## Automation tests
 - Unit tests 
   N/A
 - Integration tests
   N/A

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Samples
N/A

## Related PRs
N/A

## Migrations (if applicable)
N/A

## Test environment
N/A
 
## Learning
N/A

### CODE REVIEW

File: modules/samples/sample-clients/wso2event-client/src/main/java/org/wso2/carbon/sample/wso2event/Client.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ public static void main(String[] args) {
         DataPublisherUtil.setKeyStoreParams();
         DataPublisherUtil.setTrustStoreParams();
 
-        log.info(Arrays.deepToString(args));
+        log.info("These are the provided configurations: "+Arrays.deepToString(args));
```
Comment (ksdperera) [inline modules/samples/sample-clients/wso2event-client/src/main/java/org/wso2/carbon/sample/wso2event/Client.java]: Please verify the code formatting.

File: modules/samples/sample-clients/kafka-producer/src/main/java/org/wso2/sp/sample/kafka/client/KafkaClient.java [hunk 2]
```diff
@@ -137,9 +134,10 @@ public static void main(String[] args) {
      * @param fileName     name of the file with events
      */
     private static String getMessageFilePath(String sampleNumber, String fileName) throws Exception {
+        String fileExtension = ".txt";
```
Comment (ksdperera) [inline modules/samples/sample-clients/kafka-producer/src/main/java/org/wso2/sp/sample/kafka/client/KafkaClient.java]: Better if we can define fileExtension type as  constant in class. Any reason for moved this to method?

Comment (RAVEENSR) [reply modules/samples/sample-clients/kafka-producer/src/main/java/org/wso2/sp/sample/kafka/client/KafkaClient.java]: Intellij IDEA suggested that this field can be moved to a local variable.

Comment (RAVEENSR) [reply modules/samples/sample-clients/kafka-producer/src/main/java/org/wso2/sp/sample/kafka/client/KafkaClient.java]: Resolved in commit 72ef342

File: modules/samples/sample-clients/wso2event-client/src/main/java/org/wso2/carbon/sample/wso2event/Client.java (hunk 1 above)
Comment (RAVEENSR) [reply modules/samples/sample-clients/wso2event-client/src/main/java/org/wso2/carbon/sample/wso2event/Client.java]: Resolved in commit 72ef342

**Pull request merged**